x-pack/plugins/apm/public/components/fleet_integration/apm_policy_form/settings_definition/tail_sampling_settings.ts

```diff
@@ -0,0 +1,92 @@
+import { i18n } from '@kbn/i18n';
+import { isSettingsFormValid, OPTIONAL_LABEL } from '../settings_form/utils';
+import { PackagePolicyVars, SettingsRow } from '../typings';
+import { getDurationRt } from '../../../../../common/agent_configuration/runtime_types/duration_rt';
+
+export const TAIL_SAMPLING_ENABLED_KEY = 'tail_sampling_enabled';
+
+export function getTailSamplingSettings(): SettingsRow[] {
+  return [
+    {
+      key: TAIL_SAMPLING_ENABLED_KEY,
+      rowTitle: i18n.translate(
+        'xpack.apm.fleet_integration.settings.tailSampling.tailSamplingEnabledTitle',
+        { defaultMessage: 'Enable tail-based sampling' }
+      ),
+      rowDescription: i18n.translate(
+        'xpack.apm.fleet_integration.settings.tailSampling.enableTailSamplingDescription',
+        { defaultMessage: 'Enable tail-based sampling.' }
+      ),
+      type: 'boolean',
+      settings: [
+        {
+          key: 'tail_sampling_interval',
+          type: 'duration',
+          label: i18n.translate(
+            'xpack.apm.fleet_integration.settings.tailSampling.tailSamplingInterval',
+            {
+              defaultMessage: 'Tail sampling interval',
+            }
+          ),
+          rowTitle: i18n.translate(
+            'xpack.apm.fleet_integration.settings.tailSampling.tailSamplingIntervalTitle',
+            { defaultMessage: 'Interval' }
+          ),
+          rowDescription: i18n.translate(
+            'xpack.apm.fleet_integration.settings.tailSampling.tailSamplingIntervalDescription',
+            {
+              defaultMessage:
+                'Interval for synchronization between multiple APM Servers. Should be in the order of tens of seconds or low minutes.',
+            }
+          ),
+          labelAppend: OPTIONAL_LABEL,
+          required: false,
+          validation: getDurationRt({ min: '1s' }),
+        },
+        {
+          key: 'tail_sampling_policies',
+          type: 'yaml',
+          label: i18n.translate(
+            'xpack.apm.fleet_integration.settings.tailSampling.tailSamplingPolicies',
+            { defaultMessage: 'Tail sampling policies' }
+          ),
+          rowTitle: i18n.translate(
+            'xpack.apm.fleet_integration.settings.tailSampling.tailSamplingPoliciesTitle',
+            { defaultMessage: 'Policies' }
+          ),
+          rowDescription: i18n.translate(
+            'xpack.apm.fleet_integration.settings.tailSampling.tailSamplingPoliciesDescription',
+            {
+              defaultMessage:
+                'Policies map trace events to a sample rate. Each policy must specify a sample rate. Trace events are matched to policies in the order specified. All policy conditions must be true for a trace event to match. Each policy list should conclude with a policy that only specifies a sample rate. This final policy is used to catch remaining trace events that don’t match a stricter policy.',
+            }
+          ),
+          labelAppendLink: i18n.translate(
```

Comment:
A link to the docs will be added here https://github.com/elastic/kibana/pull/124025#issuecomment-1025921645

Reply:
Raised https://github.com/elastic/kibana/issues/124265 for this